app/controllers/projects_controller.rb

```diff
@@ -0,0 +1,108 @@
+class ProjectsController < ApplicationController
+  def index
+    @projects = Project.all
+  end
+
+  def new
+    @project = Project.new
+  end
+
+  def create
+    compnay = Company.first
+    @project = Project.new
+    @project.name = params[:project][:name]
+    @project.project_category = params[:project][:project_category]
+    @project.company = compnay 
+    if(@project.save)
+      flash[:notif] = 'Project Created Successfully'      
+    else
+      flash[:notif] = 'An Error Occured'    
+    end
+    redirect_to projects_path
+  end
+
+  def edit
+    @project = Project.find(params[:id])
+  end
+
+  def update
+    project = Project.find(params[:id])
+    project.update(project_params)
+    flash[:notif] = 'Project Updated Successfully'
+    redirect_to projects_path
+  end
+
+  def show
+    @project = Project.find(params[:id])
+  end
+
+  def destroy
+    project = Project.find(params[:id])
+    project.destroy
+    flash[:notif] = 'Project Destroyed Successfully'
+    redirect_to projects_path
+  end
+
+  def project_users
+    @project = Project.find(params[:id])
+  end
+
+  def new_team
+    @project = Project.find(params[:id])
+  end
+
+  def create_team
+    company = Company.first
+    project = Project.find(params[:id])
+    team = Team.find(params[:project][:team])
+    result  = ProjectTeam.create(project: project,team: team, company: company)
```

Comment:
There should be space after comma.
```suggestion
    result  = ProjectTeam.create(project: project, team: team, company: company)
```